lib/components/src/Zoom/ZoomIFrame.tsx

```diff
@@ -0,0 +1,67 @@
+import { Component, ReactElement } from 'react';
+import { browserSupportsCssZoom } from './Zoom';
+
+export type IZoomIFrameProps = {
+  scale: number;
+  children: ReactElement<HTMLIFrameElement>;
+  iFrameRef: React.MutableRefObject<HTMLIFrameElement>;
+  active?: boolean;
+};
+
+export default class ZoomIFrame extends Component<IZoomIFrameProps> {
```

Comment:
Small nit -- we've stopped using default exports in new code. Can you make these named exports instead?

Reply:
Looking great otherwise! 💯

Reply:
yes names exports are easier to trace and refactor 👍

Reply:
This should now be resolved! 🙂